src/dsl.rs

```diff
@@ -41,7 +41,7 @@ impl fmt::Display for ParseError {
     }
 }
 
-/// Evaluate the DSL. This tokenizes the input and presses the keys.
```

Comment:
I think "tokenizes" is correct in this instance?

Reply:
Yes, I was wrong